marlowe-runtime/src/Language/Marlowe/Runtime/Core/ScriptRegistry.hs

```diff
@@ -192,29 +381,26 @@ currentV1Scripts =
                       , datum = Nothing
                       , datumHash = Nothing
                       }
-                , script = either (error . show) id $ deserialiseFromCBOR (AsPlutusScript AsPlutusScriptV2) currentV1OpenRoleScriptCBOR
+                , script = node812V1OpenRoleScript
                 }
             )
           ]
     }
 
-currentV1MarloweScriptCBOR :: ByteString
-currentV1MarloweScriptCBOR =
```

Comment:
Could we read this from the data file? Either in a TH splice or with `unsafePerformIO`?

Reply:
In the past, I've used the `file-embed` package for stuff like this.